app.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Yaydoc",
+  "description": "Automatic Documentation Generation and Deployment Project.",
+  "keywords": [
+    "documentation",
+    "fossasia",
+    "node",
+    "sphinx"
+  ],
+  "repository": "https://github.com/fossasia/yaydoc",
+  "env": {
+    "PYPANDOC_PANDOC": {
```

Comment:
If you require the user to set an Env variable to deploy it to heroku, add those here too so that the user will be requested to set them during first deployment